_layouts/application.html

```diff
@@ -18,22 +18,22 @@
 			{{ content }}
 		</div>
 
-		<div class="footer wrap">
-			<p class="about">
+		<div class="footer wrap h-card">
+			<p class="about p-note">
 				I've been called many things:
-				<a href="http://www.github.com/tpitale">Developer</a>
-				<a href="http://www.livingsocial.com">at LivingSocial,</a>
+				<a href="http://www.github.com/tpitale" class="p-title" rel="me">Developer</a>
+				<a href="http://www.livingsocial.com">at <span class="p-org">LivingSocial</span>,</a>
 				<a href="http://www.vt.edu">Hokie,</a>
 				<a href="http://www.viget.com">Viget Lab Rat,</a>
-				<a href="http://www.flickr.com/photos/thatdbeme/">Photographer,</a>
-				<a href="http://facebook.com/tpitale">Friend,</a>
-				<a href="http://speakerrate.com/tpitale">Presenter,</a>
-				<a href="http://www.twitter.com/tpitale">Orator</a>
+				<a href="http://www.flickr.com/photos/thatdbeme/" rel="me">Photographer,</a>
+				<a href="http://facebook.com/tpitale" rel="me">Friend,</a>
+				<a href="http://speakerrate.com/tpitale" rel="me">Presenter,</a>
+				<a href="http://www.twitter.com/tpitale" rel="me">Orator</a>
```

Comment:
With these `rel="me"` attributes on links to your profiles, you can use any site that supports [IndieAuth](https://indieauth.com) to log in using your site's domain.